projects/baby_driver/scripts/test_gpio_set.py

```diff
@@ -0,0 +1,59 @@
+"""This Module Tests methods in gpio_set.py"""
+import unittest
+from unittest.mock import patch
+
+from gpio_set import gpio_set
+from gpio_port import GpioPort
+
+class TestGpioSet(unittest.TestCase):
+    """Test gpio_set function"""
+
+    @patch('can_util.send_message')
+    @patch('can_util.next_message')
+    def test_low_bound(self, mock_send_message, mock_next_message):
+        """Tests minimum values for pin and port"""
+        mock_send_message.side_effect = None
+        mock_next_message.side_effect = None
```

Comment:
`side_effect` is `None` by default, you don't have to set it explicitly; this goes for the other places you do that too